### PR TITLE
*: fix cert and env binding for k8s-node-bootstrapper

### PR DESCRIPTION
**Pull request by coresolve**
This change adds a bindmount of /etc/ssl and /etc/profile.env to ensure
that we can operate in proxy environments.

@lucab @lander2k2

### CODE REVIEW

File: modules/ignition/resources/services/k8s-node-bootstrap.service [hunk 1]
```diff
@@ -24,6 +24,8 @@ ExecStartPre=/usr/bin/docker run --rm \
             -v /etc/coreos:/etc/coreos:ro \
             -v /etc/torcx:/etc/torcx \
             -v /etc/kubernetes:/etc/kubernetes \
+            -v /etc/ssl:/etc/ssl:ro \
```
Comment (squat) [inline modules/ignition/resources/services/k8s-node-bootstrap.service]: @coresolve the k8s-notebootstrapper service already bindmounts in /etc/ssl/certs/ca-certificates.crt [0]. Do you explicitly need all of /etc/ssl? If so, you should omit the first bind mount, since it's being stomped.

[0] https://github.com/coreos/tectonic-installer/pull/2460/files#diff-dea3ad9663a5b686a1d43a0d5d7b167eR18

File: modules/ignition/resources/services/k8s-node-bootstrap.service [hunk 2]
```diff
@@ -24,6 +24,8 @@ ExecStartPre=/usr/bin/docker run --rm \
...
+            -v /etc/profile.env:/etc/profile.env:ro \
```
Comment (squat) [inline modules/ignition/resources/services/k8s-node-bootstrap.service]: JW, why is this needed?

File: modules/ignition/resources/services/k8s-node-bootstrap.service (hunk 1 above)
Comment (coresolve) [reply modules/ignition/resources/services/k8s-node-bootstrap.service]: I've pulled the other mount. We need it to support things like egress proxy where all traffic is sent to an env set proxy server.

File: modules/ignition/resources/services/k8s-node-bootstrap.service (hunk 2 above)
Comment (coresolve) [reply modules/ignition/resources/services/k8s-node-bootstrap.service]: Needed to makes sure we pull in settings like: 
```
HTTP_PROXY=thisisabadidea:3279
HTTPS_PROXY=stillbad:3279
```
etc

**Comment by coresolve** [comment]
This  has been changed to use /etc/profile.env as a --env-file. The container has no shell and probably won't eval the /etc/profile.env file.

**Comment by lucab** [comment]
@coresolve I haven't tested tectonic-torcx with any proxy, but I assume you ran this already. Do you want me to check any specific case/scenario (or just a random quick run)?

As a warning note, any proxy configurations applied here needs to be propagated to CLUO hook, otherwise cluster upgrades will get stuck into permanent network failures.

File: modules/ignition/resources/services/k8s-node-bootstrap.service (hunk 1 above)
Comment (coresolve) [reply modules/ignition/resources/services/k8s-node-bootstrap.service]: I'm going to put this back

**Comment by coresolve** [comment]
Yep. I need to close this and reopen against track-1

**Comment by squat** [comment]
@coresolve there is no /etc/profile.env on CL machines by default. Good thing we did not merge without running tests. Please see #2757. All tests are failing because systemd fails to start kubelet.service due to:
```
Jan 18 11:49:08 ip-10-0-4-128 systemd[1]: kubelet.service: Failed to load environment files: No such file or directory
Jan 18 11:49:08 ip-10-0-4-128 systemd[1]: kubelet.service: Failed to run 'start-pre' task: No such file or directory
Jan 18 11:49:08 ip-10-0-4-128 systemd[1]: Failed to start Kubelet via Hyperkube ACI.
Jan 18 11:49:08 ip-10-0-4-128 systemd[1]: kubelet.service: Unit entered failed state.
Jan 18 11:49:08 ip-10-0-4-128 systemd[1]: kubelet.service: Failed with result 'resources'.
Jan 18 11:49:18 ip-10-0-4-128 systemd[1]: kubelet.service: Service hold-off time over, scheduling restart.
Jan 18 11:49:18 ip-10-0-4-128 systemd[1]: Stopped Kubelet via Hyperkube ACI.
```
